transform/transform_test.go

```diff
@@ -60,6 +60,13 @@ func TestReverseComplement(t *testing.T) {
 	}
 }
 
+func TestComplementBaseError(t *testing.T) {
+	complementBase := ComplementBase('!')
+	if complementBase != ' ' {
```

Comment:
@Koeng101 if complement base can't find a complement for the given nucleotide what should it return? N?